### PR TITLE
fix: duplicate icons in menu after update launcher

### PR DESCRIPTION
**Pull request by kamiyadm**
remove duplicate item which in fullscreen and window list.

Log:

### CODE REVIEW

File: src/model/appsmanager.cpp [hunk 1]
```diff
@@ -522,6 +522,18 @@ void AppsManager::sortByInstallTimeOrder(ItemInfoList_v1 &processList)
     });
 }
 
+void AppsManager::removeDuplicateData(ItemInfoList_v1 &processList)
+{
+    ItemInfoList_v1 tmpList;
+    for(const auto item : processList) {
+        if (!tmpList.constants(item)) {
```
Comment (BLumia) [inline src/model/appsmanager.cpp:529]:
```suggestion
        if (!tmpList.contains(item)) {
```

这个本地测了嘛...

File: src/model/appsmanager.cpp [hunk 2]
```diff
@@ -522,6 +522,18 @@ void AppsManager::sortByInstallTimeOrder(ItemInfoList_v1 &processList)
     });
 }
 
+void AppsManager::removeDuplicateData(ItemInfoList_v1 &processList)
+{
+    ItemInfoList_v1 tmpList;
+    for(const auto item : processList) {
```
Comment (BLumia) [inline src/model/appsmanager.cpp:528]:
```suggestion
    for (const auto item : processList) {
```

File: src/model/appsmanager.cpp (hunk 1 above)
Comment (kamiyadm) [reply src/model/appsmanager.cpp:529]: 虚拟机里测过0.0

**Review by BLumia — APPROVED** [review]

**Pull request merged**